packages/hubspot/api.js

```diff
@@ -72,6 +76,31 @@ class Api extends OAuth2Requester {
         return this.authorizationUri;
     }
 
+    addJsonHeaders(options) {
```

Comment:
@MichaelRyanWebber technically we could put this in the addAuthHeaders middleware?

Also I'm pretty sure this is defaulted at least in the _post?

But it does make me think we should have some other way of doing this other than needing to override the main methods locally... thoughts on an approach? Maybe just an addHeaders method, or object on the definition?

Cc'ing in @leofmds in case he had any suggestions

Reply:
@seanspeaks addAuthHeaders doesn't seem right to me because it's not appropriate for every request, even though it isn't likely to break anything. It's not defaulted in _post (unless I'm mistaken... but it not working by default was exactly how I ended up paying attention to this)

And I definitely agree we should bring this into the Request class, to be able to add standard headers for the different HTTP methods. The thing requiring finesse to me, is how we handle overriding vs optionally adding to the headers from with the api methods. But I don't really think it's tricky, would be a straightforward feature to add. If I felt "allowed" to, I probably would have just done that instead, since this is a common issue.

Reply:
@MichaelRyanWebber probably a perfect moment for a RFC. You could even PR the majority of it if you felt like it was a quick idea and implementing was faster than just typing out.

Probably just covering the topic default header helpers in requester?

Reply:
created a todo for me for that